src/components/ReviewItem.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getReviewById } from "../reqs/apis";
+
+const ReviewItem = ({ isLoading, setIsloading }) => {
```

Comment:
isLoading and setIsloading don't have to be passed down as props - can declare the new state here -  as the loading state in this component does not interact with any other components